docs/pages/getting-started/linux-server.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Getting started
-description: Getting started with Teleport - identity-aware, multi-protocol access proxy for SSH, HTTPS, Kubernetes, and more.
+description: This tutorial will guide you through the steps needed to install and run Teleport 7.1.0 on Linux machines.
```

Comment:
Is it possible to keep  `Getting started with Teleport`?

Reply:
Yup, just updated